### PR TITLE
CHANGE: Drop support for Unity 2018.3.

### PR DESCRIPTION
**Pull request by douglas-piconi**
Drop support for Unity 2018.3

NOTE: I pulled trunk out of Yamato config so that we can stabilize tests on 2019.1

### CODE REVIEW

File: .yamato/upm-ci.yml [hunk 1]
```diff
@@ -1,7 +1,5 @@
 editors:
-  - version: 2018.3
   - version: 2019.1
-  - version: trunk
```
Comment (douglas-piconi) [inline .yamato/upm-ci.yml:4]: NOTE: I pulled trunk out of Yamato config so that we can stabilize tests on 2019.1

**Review by Rene-Damm — COMMENTED** [review]
Could we also purge all outdated #ifs as part of this change?

Think we can even kill off the entire AdvancedDropdown code. AFAIK that stuff has landed in 2019.1 and we can just use the built-in version. This way we'll get any fixes, too.

**Comment by douglas-piconi** [comment]
@Rene-Damm ok, pulled out the ifdef's     I didn't want to touch AdvancedDropdown in this PR though.

**Comment by Rene-Damm** [comment]
@douglas-piconi Ok, I'll rip out AdvancedDropdown in a follow-up PR. Should be as simple as nuking the folder and having Rider update some usings.

**Review by Rene-Damm — APPROVED** [review]

**Pull request merged**